pycbc/events/triggers.py

```diff
@@ -178,7 +178,11 @@ def loudest_triggers_from_cli(opts, coinc_parameters=None,
                 for ifo in ifos:
                     key = "/".join([ifo, p])
                     arr = statmap.get_snglfile_array_dict(p)[ifo]
-                    data[key] = arr[bin_idx][sorting][:opts.n_loudest]
+                    if "/".join(ifo, "valid") not in data:
```

Comment:
change `"/".join(ifo, "valid")` to `ifo + "/valid"` > more readable

Reply:
(A bit of explanation of this bit would be nice as well)

Reply:
Here I'm just following the style of the code surrounding this. 

What I'm doing here is recording which entries of the various single-detector quantity arrays should not be used because that detector was not active.

I'm not actually convinced this code block is used anywhere. So I will ping Alex about whether to remove this.